roofit/roofitcore/inc/RooDataHist.h

```diff
@@ -112,9 +112,9 @@ class RooDataHist : public RooAbsData, public RooDirItem {
 
   TIterator* sliceIterator(RooAbsArg& sliceArg, const RooArgSet& otherArgs) ;
 
-  void weightError(Double_t& lo, Double_t& hi, ErrorType etype=Poisson) const override;
-  /// Return the error of the weight of the last-retrieved entry. See also weightError(Double_t&,Double_t&,ErrorType) const.
-  Double_t weightError(ErrorType etype=Poisson) const override {
+  void weightError(double& lo, double& hi, ErrorType etype=Poisson) const override;
+  /// \copydoc RooAbsData::weightError(ErrorType)
```

Comment:
Pls try that this works. I *think* that you need the `const` to generate the correct link.

Reply:
In general, to get the correct link, you have to generate this string:
```
void weightError(double& lo, double& hi, ErrorType etype=Poisson) const override;
weightError(double&,double&,ErrorType) const
```
That is: Types (incl. const and ref/pointer) of the arguments, and the function signature without return types.
Class name is optional when inside the class.